sources/settings-migrations/v1.28.0/kernel-sysctl-hugepages/src/main.rs

```diff
@@ -0,0 +1,21 @@
+use migration_helpers::common_migrations::{AddMetadataMigration, SettingMetadata};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+///  We added new settings metadata, `metadata.settings.kernel.sysctl.vm/nr_hugepages.setting-generator`
+fn run() -> Result<()> {
+    migrate(AddMetadataMigration(&[SettingMetadata {
+        metadata: &["setting-generator"],
+        setting: "metadata.settings.kernel.sysctl.vm/nr_hugepages",
```

Comment:
I think you want to provide the setting itself here:
```suggestion
        setting: "settings.kernel.sysctl.vm/nr_hugepages",
```